Lib/test/test_gzip.py

```diff
@@ -491,6 +491,16 @@ def test_compress(self):
                 with gzip.GzipFile(fileobj=io.BytesIO(datac), mode="rb") as f:
                     self.assertEqual(f.read(), data)
 
+    def test_compress_mtime(self):
+        mtime = 123456789
+        for data in [data1, data2]:
+            for args in [(1, mtime), (6, mtime), (9, mtime)]:
```

Comment:
maybe you could use the [subTest](https://docs.python.org/3.8/library/unittest.html?highlight=subtest#distinguishing-test-iterations-using-subtests) features.